cfpb/cfgov-refresh
# Convert social media icons to use SVG icons

Spins social media SVG icon updates from https://github.com/cfpb/cfgov-refresh/pull/3887.

## Changes

- Converts social media icons to use SVG icons.
- Updates code comments to use SVG.

## Testing

1. Visit /blog and click an article. Inspect the sharing icons and see that they are SVG.
2. Visit /practitioner-resources/your-money-your-goals/, click play on the video and inspect the sharing icons and see that they are SVG (transparency on the icon gives it away too).

## Screenshots

![screen shot 2018-05-16 at 10 52 18 am](https://user-images.githubusercontent.com/704760/40124806-56a5c306-58f7-11e8-8c67-d10039a5ba1a.png)

![screen shot 2018-05-16 at 10 53 33 am](https://user-images.githubusercontent.com/704760/40124842-6882e72a-58f7-11e8-82af-d6478ac01fed.png)
